src/module.ts

```diff
@@ -193,48 +194,48 @@ export enum UnaryOp {
-  AbsF32x4 = 85 /* _BinaryenAbsVecF32x4 */,
-  NegF32x4 = 86 /* _BinaryenNegVecF32x4 */,
-  SqrtF32x4 = 87 /* _BinaryenSqrtVecF32x4 */,
-  CeilF32x4 = 88 /* _BinaryenCeilVecF32x4 */,
-  FloorF32x4 = 89 /* _BinaryenFloorVecF32x4 */,
+  NegI16x8 = 74 /* _BinaryenNegVecI16x8 */,
+  AnyTrueI16x8 = 75 /* _BinaryenAnyTrueVecI16x8 */,
+  AllTrueI16x8 = 76 /* _BinaryenAllTrueVecI16x8 */,
+  BitmaskI16x8 = 77 /* _BinaryenBitmaskVecI16x8 */,
+  AbsI32x4 = 78 /* _BinaryenAbsVecI32x4 */,
+  NegI32x4 = 79 /* _BinaryenNegVecI32x4 */,
+  AnyTrueI32x4 = 80 /* _BinaryenAnyTrueVecI32x4 */,
+  AllTrueI32x4 = 81 /* _BinaryenAllTrueVecI32x4 */,
+  BitmaskI32x4 = 82 /* _BinaryenBitmaskVecI32x4 */,
+  NegI64x2 = 83 /* _BinaryenNegVecI64x2 */,
+  AnyTrueI64x2 = 84 /* _BinaryenAnyTrueVecI64x2 */,
+  AllTrueI64x2 = 85 /* _BinaryenAllTrueVecI64x2 */,
+  AbsF32x4 = 86 /* _BinaryenAbsVecF32x4 */,
+  NegF32x4 = 87 /* _BinaryenNegVecF32x4 */,
+  SqrtF32x4 = 88 /* _BinaryenSqrtVecF32x4 */,
+  CeilF32x4 = 89 /* _BinaryenCeilVecF32x4 */,
+  FloorF32x4 = 90 /* _BinaryenFloorVecF32x4 */,
   TruncF32x4 = 90 /* BinaryenTruncVecF32x4 */,
   NearestF32x4 = 91 /* BinaryenNearestVecF32x4 */,
-  AbsF64x2 = 92 /* _BinaryenAbsVecF64x2 */,
-  NegF64x2 = 93 /* _BinaryenNegVecF64x2 */,
-  SqrtF64x2 = 94 /* _BinaryenSqrtVecF64x2 */,
-  CeilF64x2 = 95 /* _BinaryenCeilVecF64x2 */,
-  FloorF64x2 = 96 /* _BinaryenFloorVecF64x2 */,
+  AbsF64x2 = 93 /* _BinaryenAbsVecF64x2 */,
+  NegF64x2 = 94 /* _BinaryenNegVecF64x2 */,
+  SqrtF64x2 = 95 /* _BinaryenSqrtVecF64x2 */,
+  CeilF64x2 = 96 /* _BinaryenCeilVecF64x2 */,
+  FloorF64x2 = 97 /* _BinaryenFloorVecF64x2 */,
   TruncF64x2 = 97 /* BinaryenTruncVecF64x2 */,
   NearestF64x2 = 98 /* BinaryenNearestVecF64x2 */,
-  TruncSatF32x4ToI32x4 = 99 /* _BinaryenTruncSatSVecF32x4ToVecI32x4 */,
-  TruncSatF32x4ToU32x4 = 100 /* _BinaryenTruncSatUVecF32x4ToVecI32x4 */,
-  TruncSatF64x2ToI64x2 = 101 /* _BinaryenTruncSatSVecF64x2ToVecI64x2 */,
-  TruncSatF64x2ToU64x2 = 102 /* _BinaryenTruncSatUVecF64x2ToVecI64x2 */,
-  ConvertI32x4ToF32x4 = 103 /* _BinaryenConvertSVecI32x4ToVecF32x4 */,
-  ConvertU32x4ToF32x4 = 104 /* _BinaryenConvertUVecI32x4ToVecF32x4 */,
-  ConvertI64x2ToF64x2 = 105 /* _BinaryenConvertSVecI64x2ToVecF64x2 */,
-  ConvertU64x2ToF64x2 = 106 /* _BinaryenConvertUVecI64x2ToVecF64x2 */,
-  WidenLowI8x16ToI16x8 = 107 /* _BinaryenWidenLowSVecI8x16ToVecI16x8 */,
-  WidenHighI8x16ToI16x8 = 108 /* _BinaryenWidenHighSVecI8x16ToVecI16x8 */,
-  WidenLowU8x16ToU16x8 = 109 /* _BinaryenWidenLowUVecI8x16ToVecI16x8 */,
-  WidenHighU8x16ToU16x8 = 110 /* _BinaryenWidenHighUVecI8x16ToVecI16x8 */,
-  WidenLowI16x8ToI32x4 = 111 /* _BinaryenWidenLowSVecI16x8ToVecI32x4 */,
-  WidenHighI16x8ToI32x4 = 112 /* _BinaryenWidenHighSVecI16x8ToVecI32x4 */,
-  WidenLowU16x8ToU32x4 = 113 /* _BinaryenWidenLowUVecI16x8ToVecI32x4 */,
-  WidenHighU16x8ToU32x4 = 114 /* _BinaryenWidenHighUVecI16x8ToVecI32x4 */
+  TruncSatF32x4ToI32x4 = 100 /* _BinaryenTruncSatSVecF32x4ToVecI32x4 */,
```

Comment:
And here

Reply:
Also strange: https://github.com/WebAssembly/binaryen/blob/68294338a1cc7337e808671e75933b1134d18a90/src/wasm.h#L191-L194